src/Grpc.AspNetCore.Server/IGrpcServiceActivator.cs

```diff
@@ -19,13 +19,13 @@
 using System;
 using System.Threading.Tasks;
 
-namespace Grpc.AspNetCore.Server.Internal
+namespace Grpc.AspNetCore.Server
 {
     /// <summary>
     /// A <typeparamref name="TGrpcService"/> activator abstraction.
     /// </summary>
     /// <typeparam name="TGrpcService">The service type.</typeparam>
-    internal interface IGrpcServiceActivator<TGrpcService> where TGrpcService : class
+    public interface IGrpcServiceActivator<TGrpcService> where TGrpcService : class
```

Comment:
Does this (and other activator related class) need to be published?

Do we have confidence that these interfaces are stable and won't change?

Reply:
I rebased it on top of this PR - https://github.com/grpc/grpc-dotnet/pull/677

The activators were originally public. We made them internal just prior to 1.0 because no one had asked for them. Now some people are asking for them: https://github.com/grpc/grpc-dotnet/issues/671